src/main/webapp/WEB-INF/views/user/register-kid.jsp

```diff
@@ -102,11 +102,17 @@
 </form:form>
 <script src="http://cdn.jsdelivr.net/jquery.validation/1.15.0/jquery.validate.min.js"></script>
 <script src="http://cdn.jsdelivr.net/jquery.validation/1.15.0/additional-methods.min.js"></script>
+<script src="resources/js/validation-kids.js"></script>
+
 <c:if test="${pageContext.response.locale=='ua'}">
```

Comment:
I think that it`ll be better to use "c: choose" tag. Because when we will add a lot of new languages it will be problem with lots of "c if" tags.